### PR TITLE
Fix compilation error with Qt 5.15

### PR DESCRIPTION
**Pull request by data-man**

### CODE REVIEW

**Comment by pbek** [comment]
Thank you very much.

**Comment by pbek** [comment]
#1666

File: qmarkdowntextedit.cpp [hunk 1]
```diff
@@ -22,6 +22,9 @@
 #include <QKeyEvent>
 #include <QLayout>
 #include <QPainter>
+#if QT_VERSION >= QT_VERSION_CHECK(5, 15, 0)
```
Comment (Waqar144) [inline qmarkdowntextedit.cpp]: I dont think the #ifdefs are necessary.

Comment (pbek) [reply qmarkdowntextedit.cpp]: Possibly...

Comment (data-man) [reply qmarkdowntextedit.cpp]: After upgrading Qt I skipped many versions of QOwnNotes from AUR.
So I decided made the patch.

Comment (bjorn) [reply qmarkdowntextedit.cpp:25]: Right, but this is not a new header, so the `#if` is not needed.

Comment (data-man) [reply qmarkdowntextedit.cpp:25]: @bjorn #113 ;)

**Pull request merged**